src/test/java/seedu/resireg/logic/commands/HistoryCommandTest.java

```diff
@@ -14,22 +14,27 @@
     private Model model = new ModelManager();
     private Model expectedModel = new ModelManager();
 
+    private final String labelOne = "1" + "\t";
+    private final String labelTwo = "2" + "\t";
+    private final String labelThree = "3" + "\t";
```

Comment:
As above, these labels should be placed in a constants file. `TypicalCommandHistories` works as well, especially if we're looking to add a `CommandHistoryBuilder` or add integration tests in the future.

Reply:
In line with what has been mentioned above, I'll place these labels in a constants file.